src/plugins/dashboard/public/dashboard_listing/hooks/use_dashboard_listing_table.tsx

```diff
@@ -42,6 +42,7 @@ const toTableListViewSavedObject = (hit: DashboardItem): DashboardSavedObjectUse
     type: 'dashboard',
     id: hit.id,
     updatedAt: hit.updatedAt!,
+    namespaces: hit.namespaces!,
```

Comment:
As with `updatedAt`, the `namespaces` property is always present on dashboard saved objects, so we add the non-null assertion.